components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -98,75 +106,156 @@ const WorkspacesPage: FunctionComponent = () => {
-                        />
-                        <ItemsList className="app-container pb-40">
-                            <div className="border-t border-gray-200 dark:border-gray-800"></div>
-                            {filteredActiveWorkspaces.map((info) => {
-                                return <WorkspaceEntry key={info.id} info={info} />;
-                            })}
-                            {filteredActiveWorkspaces.length > 0 && <div className="py-6"></div>}
-                            {filteredInactiveWorkspaces.length > 0 && (
-                                <div>
-                                    <div
-                                        onClick={() => setShowInactive(!showInactive)}
-                                        className="flex cursor-pointer py-6 px-6 flex-row text-gray-400 bg-gray-50  hover:bg-gray-100 dark:bg-gray-800 dark:hover:bg-gray-700 rounded-xl mb-2"
-                                    >
-                                        <div className="pr-2">
-                                            <Arrow direction={showInactive ? "down" : "right"} />
-                                        </div>
-                                        <div className="flex flex-grow flex-col ">
-                                            <div className="font-medium text-gray-500 dark:text-gray-200 truncate">
-                                                <span>Inactive Workspaces&nbsp;</span>
-                                                <span className="text-gray-400 dark:text-gray-400 bg-gray-200 dark:bg-gray-600 rounded-xl px-2 py-0.5 text-xs">
-                                                    {filteredInactiveWorkspaces.length}
-                                                </span>
-                                            </div>
-                                            <div className="text-sm flex-auto">
-                                                Workspaces that have been stopped for more than 24 hours. Inactive
-                                                workspaces are automatically deleted after 14 days.{" "}
-                                                <a
-                                                    target="_blank"
-                                                    rel="noreferrer"
-                                                    className="gp-link"
-                                                    href="https://www.gitpod.io/docs/configure/workspaces/workspace-lifecycle#workspace-deletion"
-                                                    onClick={(evt) => evt.stopPropagation()}
-                                                >
-                                                    Learn more
-                                                </a>
+                        <div className={isGitpodIo() ? "!pl-0 app-container flex flex-row w-full" : "app-container"}>
+                            <div>
```

Comment:
`className="flex-1"` will make this part use up the space

Reply:
Resolved in [`e7e8772`](https://github.com/gitpod-io/gitpod/pull/20021/commits/e7e8772a7f7a3c535345d8884c63947a579574fd)